src/blocks/scratch3_motion.js

```diff
@@ -254,6 +259,8 @@ class Scratch3MotionBlocks {
     getDirection (args, util) {
         return util.target.direction;
     }
+
+    doNothing () {}
```

Comment:
I sort of dislike putting this here, because there are some other hacked blocks - in different categories - that do nothing, and I'll have to put a `doNothing() {}` function in those categories too. Ideally this might go on a superclass, but `Scratch3MotionBlocks` (and all the other classes) don't have a common (or any) superclass! But not a big deal; just a sad, nothing-to-be-done nitpick.

Reply:
@towerofnix, see [my comment](https://github.com/LLK/scratch-vm/pull/1102/files#r185251022) on your other PR. I'd prefer just explicitly setting all the no-op blocks to `() => {}` directly with a short line comment explaining that these are legacy no-op blocks.